src/main/scala/cromwell/engine/backend/ExecutionResult.scala

```diff
@@ -21,8 +21,13 @@ final case class SuccessfulBackendCallExecution(outputs: CallOutputs, executionE
  */
 case object AbortedExecution extends ExecutionResult
 
-/**
- * Failed execution, possibly having a return code.
- */
-final case class FailedExecution(e: Throwable, returnCode: Option[Int] = None) extends ExecutionResult
+sealed trait FailedExecution extends ExecutionResult {
+  def e: Throwable
```

Comment:
Is this really an exception or are we just after the error message? i.e. is the actual exception type meaningful elsewhere?